cxx-sensors/src/main/java/org/sonar/cxx/sensors/visitors/CxxFileLinesVisitor.java

```diff
@@ -74,8 +78,9 @@
    * @param fileLinesContextFactory container for linesOfCode, linesOfComments, executableLines
    * @param allLinesOfCode set of lines for a source file
    */
-  public CxxFileLinesVisitor(FileLinesContextFactory fileLinesContextFactory, SensorContext context,
+  public CxxFileLinesVisitor(CxxLanguage language, FileLinesContextFactory fileLinesContextFactory, SensorContext context,
```

Comment:
I do not think that the CxxLanguage (C++, C) is helpful to fix this issue.

Reply:
Helper CxxUtils.validateRecovery(e, language); needs it.